railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -14,6 +14,11 @@ Rails.application.configure do
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
+  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
+  # `config/secrets.yml.key`.
+  config.read_encrypted_secrets = true
```

Comment:
We skirted away from silently reading them in development just because users have a key and it's not our recommendation to use encrypted secrets as dev secrets storage.